part-two/Attention.md

```diff
@@ -15,9 +15,11 @@ But is any of that really a problem? Is attention scarce in the precise meaning
 
-In combination this is leading to a large scale crisis of individual identity. This crisis takes many different forms, including increased teenage depression, growing adult suicide rates (especially among middle-aged white males), drug overdose deaths and rising aggression both online and offline {{"NYT2" | cite}}. This is not dissimilar from the beginning of the Industrial Age, when people had to leave the countryside and move to big cities. They were forced to give up identities that had been constructed around land and a historical set of professions. They were confronted with people from other regions who held different beliefs. 
+In combination this is leading to a large scale crisis of individual identity. This crisis takes many different forms, including increased teenage depression, growing adult suicide rates (especially among middle-aged white males), drug overdose deaths and rising aggression both online and offline {{"TIME" | cite}} {{"NPR2" | cite}} {{"NYT2" | cite}}. This is not dissimilar from the beginning of the Industrial Age, when people had to leave the countryside and move to big cities. They were forced to give up identities that had been constructed around land and a historical set of professions. They were confronted with people from other regions who held different beliefs. 
 
-And so it should not be at all surprising that there is a rise in populist leaders with simplistic messages. People whose identity is shaken want to be reassured. They want to hear that things will be OK and that the way of getting there is simple. &ldquo;Make America Great Again&rdquo; is an example of that. So is ISIS. In both cases the message is retrograde. Instead of a new identity that has to be built, requiring time and effort, these backward movements promise an easy return to a glorious identity of the past.  
+And so it should not be at all surprising that there is a rise in populist leaders with simplistic messages, such as Donald Trump in the United States and Marine Le Pen in France. A recent study has found that throughout Europe, populist parties are receiving more than double their average share of the vote in national and parliamentary elections since the 1960s {{"HARV2" |cite}}. 
+
+At the same time, there have been increasing attempts to suppress free speech among groups and organizations with conflicting belief systems. This has been particularly evident on college campuses throughout the United States {{"CNN" | cite}}. People whose identity is shaken want to be reassured. They want to hear that things will be OK and that the way of getting there is simple. &ldquo;Make America Great Again&rdquo; is an example of that. So is ISIS. In both cases the message is retrograde. Instead of a new identity that has to be built, requiring time and effort, these backward movements promise an easy return to a glorious identity of the past.  
```

Comment:
I don't really want to wade  into the College PC debate with a side sentence here. Please remove the two sentences "At the same time ... " and "This has been ..."